pkg/config/setup/config.go

```diff
@@ -289,6 +289,7 @@ func InitConfig(config pkgconfigmodel.Setup) {
 	config.BindEnvAndSetDefault("check_sampler_expire_metrics", true)
 	config.BindEnvAndSetDefault("check_sampler_context_metrics", false)
 	config.BindEnvAndSetDefault("host_aliases", []string{})
+	config.BindEnvAndSetDefault("host_disable_ccrid_fetch", false)
```

Comment:
We have a bunch of `collect_XXX` settings for the metadata (`collect_gce_tags`, `collect_ec2_tags`, `collect_ec2_instance_info`, `collect_gpu_tags`). What do you think about `collect_ccrid` instead ?

We can change this in a later PR if needed.